docs/en/03_examples/00_elementalarea_with_dataobject.md

```diff
@@ -62,11 +52,13 @@ class BlogPost extends DataObject
  ```
 
 If you are using `ElementalArea` together with `DataObject`, it is important to define the `CMSEditLink()` method in the class.\
-`BaseElement::CMSEditLink()` relies on a valid CMS link being available in the parent `DataObject` - in this case, `BlogPost`. It is used to navigate directly to the editing section of the particular element.\
-***Note:*** For nested `GridField`s this method can get more complicated. Similarly, if your class is used in multiple admins, you will have to choose one to be the canonical admin section for the purposes of this method. This example only shows the simplest case, where the `BlogPost` class is used directly in `BlogPostsAdmin`.
+[`BaseElement::CMSEditLink()`](api:DNADesign\Elemental\Models\BaseElement::CMSEditLink()) relies on a valid CMS link being available in the parent `DataObject` - in this case, `BlogPost`. It is used to navigate directly to the editing section of the particular element.
+
+> [!NOTE]
+> If you have a nested [`GridField`](api:SilverStripe\Forms\GridField\GridField) this method can get more complicated. Similarly, if your class is used in multiple admins, you will have to choose one to be the canonical admin section for the purposes of this method. This example only shows the simplest case, where the `BlogPost` class is used directly in `BlogPostsAdmin`.
 
 ```php
-// ...
+namespace App\Models;
 
 class BlogPost extends DataObject
```

Comment:
Eventually we should probably remove this example and link to [Getting an edit link](https://docs.silverstripe.org/en/5/developer_guides/model/managing_records/#getting-an-edit-link) - you can do that now if you like or we can kick it down the road.

Reply:
Do it later